aml/src/expression.rs

```diff
@@ -350,6 +405,26 @@ where
         .map(|((), result)| Ok(result))
 }
 
+fn def_l_not<'a, 'c>() -> impl Parser<'a, 'c, AmlValue>
+where
+    'c: 'a,
+{
+    /*
+     * DefLNot := 0x92 Operand
+     * Operand := TermArg => Integer
+     */
+    opcode(opcode::DEF_L_NOT_OP)
+        .then(comment_scope(
+            DebugVerbosity::AllScopes,
+            "DefLNot",
+            term_arg().map_with_context(|arg, context| {
+                let operand = try_with_context!(context, arg.as_bool());
+                (Ok(AmlValue::Boolean(operand)), context)
```

Comment:
I might just be missing it, but I don't see where the actual not is happening. Should this be `!operand`?